js/app/modules/arrangement/constrained.js

```diff
@@ -109,7 +109,7 @@ var Constrained = new Module.Class({
             0, 0,
             widgets,
             {});
-        constraints.forEach(this.add_constraint, this);
+        constraints.forEach(c => { this.add_constraint (c); }, this);
```

Comment:
nitpick: Can be done without the extra `this` binding since the arrow function binds it already.
`constraints.forEach(c => this.add_constraint(c));`